modules/overview-of-kueue-resources.adoc

```diff
@@ -55,53 +66,28 @@ spec:
     effect: "NoSchedule"
 
 ----
-////
+
+Make sure that each flavor has the correct label selectors and taint tolerations so that workloads run on the expected nodes.
```

Comment:
```suggestion
Make sure that each resource flavor has the correct label selectors and taint tolerations so that workloads run on the expected nodes.
```
Perhaps for a bit more clarity?